inferelator_ng/design_response_R.py

```diff
@@ -3,14 +3,11 @@
 """
 
 import os
-import subprocess
```

Comment:
I'm confused why this is showing up in this pull request diff, since these changes have already been made on the master branch: https://github.com/simonsfoundation/inferelator_ng/commit/bbf2ca4b316d567ad2bddb1088028e67e31d18fe